LibVLCSharp/Shared/Events/EventManager.cs

```diff
@@ -11,16 +10,38 @@ internal abstract class EventManager
         internal readonly struct Native
         {
             [DllImport(Constants.LibraryName, CallingConvention = CallingConvention.Cdecl, EntryPoint = "libvlc_event_attach")]
-            internal static extern int LibVLCEventAttach(IntPtr eventManager, EventType eventType, EventCallback eventCallback,
+            internal static extern int LibVLCEventAttach(IntPtr eventManager, EventType eventType, InternalEventCallback eventCallback,
                 IntPtr userData);
 
             [DllImport(Constants.LibraryName, CallingConvention = CallingConvention.Cdecl, EntryPoint = "libvlc_event_detach")]
-            internal static extern void LibVLCEventDetach(IntPtr eventManager, EventType eventType, EventCallback eventCallback,
+            internal static extern void LibVLCEventDetach(IntPtr eventManager, EventType eventType, InternalEventCallback eventCallback,
                 IntPtr userData);
         }
 
+        /// <summary>
+        /// The class that manages one type of event
+        /// </summary>
+        internal class EventTypeManager: IDisposable
+        {
+            internal Action<IntPtr> EventHandler { get; }
+            internal GCHandle _handle;
+
+            internal EventTypeManager(Action<IntPtr> eventHandler)
+            {
+                EventHandler = eventHandler;
+                _handle = GCHandle.Alloc(this);
```

Comment:
Guess you can use the same casing here since they're both `internal`

Reply:
Yes, that should be a property instead of a field